README.md

```diff
@@ -1,11 +1,123 @@
-# MSAL Native Auth Sample App Private Preview
+# Sign in users and call a protected web API in iOS (Swift) mobile app by using native authentication
 
-This is a sample application used to demonstrate the CIAM extension of the MSAL framework that is currently in public preview. 
+* [Overview](#overview)
+* [Contents](#contents)
+* [Prerequisites](#prerequisites)
+* [Project setup](#project-setup)
+* [Key concepts](#key-concepts)
+* [Reporting problems](#reporting-problems)
+* [Contributing](#contributing)
 
-## Prerequisite
+## Overview
 
-To install and run the sample application a Mac with Xcode installed is required.
+This sample iOS sample applications demonstrates how to handle sign-up, sign-in, sign out, and reset password scenarios using Microsoft Entra External ID for customers. You can configure the sample to call a protected web API.
 
-## Setup
-After you have downloaded the code or cloned the repo, you need to configure the app.
-Update the `Configuration.swift` file with your CIAM client ID and CIAM tenant subdomain.
\ No newline at end of file
+## Contents
+
+| File/folder | Description |
```

Comment:
the "[NativeAuthSampleApp.xcodeproj](https://github.com/Azure-Samples/ms-identity-ciam-native-auth-ios-sample/tree/main/NativeAuthSampleApp.xcodeproj)" is still missing from this list